examples/CRISP/client/.env.example

```
@@ -1,3 +1,5 @@
 VITE_ENCLAVE_API=http://127.0.0.1:4000
-VITE_TWITTER_SERVERLESS_API=
 VITE_WALLETCONNECT_PROJECT_ID=
+# A token with a public mint function. Also used as the fee token for Enclave. 
+# This is its default address in Hardhat node
+VITE_CRISP_TOKEN="0x9fE46736679d2D9a65F0992F2272dE9f3c7fa6e0"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)